spec/models/user_spec.rb

```diff
@@ -151,30 +206,31 @@
       webauth_privgroup_str = 'sdr:administrator-role|sdr:manager-role|sdr:viewer-role'
       mock_webauth = double('webauth', :login => 'asdf', :logged_in? => true, :privgroup => webauth_privgroup_str)
       user = User.find_or_create_by_webauth(mock_webauth)
-      expect(user.is_admin  ).to be_truthy
-      expect(user.is_manager).to be_truthy
-      expect(user.is_viewer ).to be_truthy
-      user.set_groups_to_impersonate(['workgroup:sdr:not-an-administrator-role', 'workgroup:sdr:not-a-manager-role', 'workgroup:sdr:not-a-viewer-role'])
-      expect(user.is_admin  ).to be_falsey
-      expect(user.is_manager).to be_falsey
-      expect(user.is_viewer ).to be_falsey
+      expect(user.is_admin).to be true
```

Comment:
I prefer truthiness in the same way I prefer `be_nil` to `be(nil)`.  At some point it was recommended by transpec or one of the tools. It isn't a big deal though.  

This test was previously testing before/after impersonation.  Now it isn't.  That seems weird.